js/public-share.js

```diff
@@ -53,11 +53,7 @@ $(document).ready(function(){
 				.html(DOMPurify.sanitize(
 					marked(content, {
 						renderer: renderer,
-						gfm: false,
```

Comment:
This is the only non-default value here. Default is `true` and thus it fixes the stuff in here.